### PR TITLE
Added ModifyApplicationClassLoader procedure

### PR DESCRIPTION
**Pull request by justnoxx**

### CODE REVIEW

File: src/main/resources/project/parameterForms/ModifyApplicationClassLoader.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        <propertyReference>/plugins/@PLUGIN_NAME@/project/websphere_cfgs</propertyReference>
+        <required>1</required>
+    </formElement>
+    <formElement>
+		<type>entry</type>
+        <label>Wsadmin Absolute Path:</label>
+		<property>wsadminAbsPath</property>
+		<required>1</required>
+        <documentation>Path to wsadmin. If the location of wsadmin is in the PATH environment variable, then just the script name can be specified omitting its full path. E.g. '/path/to/wsadmin.sh' or 'wsadmin.bat'.</documentation>
+        <propertyReference type="property">/plugins/@PLUGIN_NAME@/project/ec_discovery/discovered_data/$configurationName/wsadminPath</propertyReference>
```
Comment (justnoxx) [inline src/main/resources/project/parameterForms/ModifyApplicationClassLoader.xml]: Done

File: src/main/resources/project/parameterForms/ModifyApplicationClassLoader.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+
+-->
+<editor>
+    <help>/commander/pages/@PLUGIN_NAME@/@PLUGIN_KEY@_help?s=Administration&amp;ss=Plugins#MapSharedLibrary</help>
```
Comment (justnoxx) [inline src/main/resources/project/parameterForms/ModifyApplicationClassLoader.xml]: Done

File: src/main/resources/project/parameterForms/ModifyApplicationClassLoader.xml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        <option><name>SINGLE</name><value>SINGLE</value></option>
+        <option><name>MULTIPLE</name><value>MULTIPLE</value></option>
+    </formElement>
+
+    <formElement>
+		<type>entry</type>
+		<label>Application Name:</label>
+		<property>applicationName</property>
+		<required>1</required>
+		<documentation>Name of application to which shared library will be mapped.</documentation>
```
Comment (justnoxx) [inline src/main/resources/project/parameterForms/ModifyApplicationClassLoader.xml]: Done

File: src/main/resources/project/parameterForms/ModifyApplicationClassLoader.xml (hunk 1 above)
Comment (imago-storm) [inline src/main/resources/project/parameterForms/ModifyApplicationClassLoader.xml]: Nope, there is still the word "absolute" in the label. And in the property name (in fact I guess we will not change that)

File: src/main/resources/project/project.xml [hunk 4]
```diff
@@ -617,7 +621,150 @@
...
+                <formalParameterName>classLoaderPolicy</formalParameterName>
+                <defaultValue />
+                <description>ClassLoader policy</description>
+                <required>0</required>
+                <type>select</type>
+            </formalParameter>
+            <formalParameter>
+                <formalParameterName>wsadminAbsPath</formalParameterName>
+                <defaultValue />
+                <description>Absolute path of wsadmin, including filename and extension. i.e: '/path/to/wsadmin.sh' or 'wsadmin.bat'. </description>
```
Comment (imago-storm) [inline src/main/resources/project/project.xml]: And here, wsadmin.bat is clearly not an absolute path.

File: src/main/resources/project/project.xml [hunk 5]
```diff
@@ -617,7 +621,150 @@
...
+                <required>1</required>
+                <type>entry</type>
+            </formalParameter>
+            <step>
+                <stepName>ModifyApplicationClassLoader</stepName>
+                <alwaysRun>0</alwaysRun>
+                <broadcast>0</broadcast>
+                <command></command>
+                <condition/>
+                <description>Map Shared Library</description>
```
Comment (imago-storm) [inline src/main/resources/project/project.xml]: Copypaste

**Pull request merged**